pom.xml

```diff
@@ -158,6 +158,7 @@
                     <excludes>
                         <exclude>**/test/**</exclude>
                         <exclude>**/tests/**</exclude>
+                        <exclude>**/*Constants.*</exclude>
```

Comment:
@lchayoun I added the `Constants` class to the coverage exclusion list, I don't believe we gonna have a lot of gains testing it, if you think otherwise we can roll it back later.